hawk/app/assets/stylesheets/shared/_misc.scss

```diff
@@ -32,3 +32,30 @@
     border-radius: 20px;
   }
 }
+
+.diff_sub {
+  background: #fdd;
+}
+
+.diff_add {
+  background: #dfd;
+}
+
+.diff_chg {
+  background: #ddf;
+}
+
+
+td.diff_header {
+color: #fff;
+}
+
+table.diff {
+ width: 100%;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>Properties should be ordered border, font-family, width